scripts/building/run_coverage_tests.py

```diff
@@ -71,8 +71,6 @@
     pytest_command += [
         "-n",
         args.n,
-        "--dist",
```

Comment:
what's --dist?

Reply:
pytest-xdist module, run pytest in multi-process. --dist will split test in different ways.